hub/saml-engine/src/main/java/uk/gov/ida/hub/samlengine/SamlEngineModule.java

```diff
@@ -414,7 +414,7 @@ private String getHubEntityId(SamlEngineConfiguration configuration) {
     @Provides
     @Singleton
     private ResponseFromCountryValidator getResponseFromCountryValidator() {
-        return new ResponseFromCountryValidator(new SamlStatusToIdpIdaStatusMappingsFactory());
+        return new ResponseFromCountryValidator();
```

Comment:
Same as above... I don't think there's a need have this in the binding module. We can just get whatever needs it to new one up.

Reply:
OK we'll remove this in #151